src/ConfigFactory.Avalonia.Demo/Models/DemoConfig.cs

```diff
@@ -78,4 +79,16 @@ public static ObservableCollection<string> GetThings()
             return value is true;
         });
     }
+
+    /// <summary>
+    /// Example implementation of internationalization and localization using the .resx Embedded Resource
+    /// </summary>
+    public override string Translate(string input)
+    {
+
+        if (string.IsNullOrWhiteSpace(input))
+            return input;
```

Comment:
If statements should always have brackets

```suggestion
        if (string.IsNullOrWhiteSpace(input)) {
            return input;
        }
```

Reply:
Fixed